setup.py

```diff
@@ -7,15 +7,26 @@
 adding support for audit logging, record locking, and time travel!
 """
 from setuptools import setup, find_packages
+import sys
 
-__version__ = "0.2.4"
+__version__ = "2.0.0"
+
+
+if sys.version_info[0] < 3:
+    sys.stderr.write("Requires Python 3 or up\n")
+    sys.exit(1)
 
 
 setup(
     name="chrononaut",
     version=__version__,  # noqa
     packages=find_packages(exclude=["*test*"]),
-    install_requires=["Flask>=0.12", "Flask-SQLAlchemy>=2.2", "pytz>=2017.2", "psycopg2>=2.7.1"],
+    install_requires=[
+        "Flask>=0.12",
```

Comment:
we can probably require a more recent version of flask, are we even testing it with 0.12?